AcadeMeData/tests/test_msgtags.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+from AcadeMeData.models import MessageTags, Messages, User
+
+
+@pytest.fixture
+def user_example():
+    user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                 'university': "RU",
+                 'degree': "CS"}
+    user = User.create_user(*user_data)
+    return user
+
+
+@pytest.mark.django_db
+class TestMessageTagsModel:
+    def test_get_msg_tag(self, user_example, id=1, msgID=None):
+        userID = user_example
+        msgID = Messages.create_message(id, userID, text='bla bla')
```

Comment:
Use a fixture here for create message like you did earlier
What are you testing here?

Reply:
try to think what tags do? and how would you like to use it,
then write a test